cluster-autoscaler/cloudprovider/azure/azure_manager_test.go

```diff
@@ -57,10 +67,110 @@ func TestCreateAzureManagerValidConfig(t *testing.T) {
 		AADClientSecret:     "fakeId",
 		VmssCacheTTL:        60,
 		MaxDeploymentsCount: 8,
+		CloudProviderRateLimitConfig: providerazure.CloudProviderRateLimitConfig{
+			RateLimitConfig: azclients.RateLimitConfig{
+				CloudProviderRateLimit:            false,
+				CloudProviderRateLimitBucket:      5,
+				CloudProviderRateLimitBucketWrite: 5,
+				CloudProviderRateLimitQPS:         1,
+				CloudProviderRateLimitQPSWrite:    1,
+			},
+			RouteRateLimit: &azclients.RateLimitConfig{
```

Comment:
let's remove the unused rate limit config here, e.g. route, LB, subnet, interface and so on